include/fix8/field.hpp

```diff
@@ -873,7 +873,10 @@ inline time_t time_to_epoch (const tm& ltm, int utcdiff=0)
    };
 
    const int tyears(ltm.tm_year ? ltm.tm_year - 70 : 0); // tm->tm_year is from 1900.
-   const int tdays(mon_days[ltm.tm_mon] + (ltm.tm_mday ? ltm.tm_mday - 1 : 0) + tyears * 365 + (tyears + 2) / 4);
+   int leaps((tyears + 2) / 4);
```

Comment:
is there any reason we havent taken solution to this directly from fix8 fix?

Reply:
This fix isn't quite right. It should be a faster version of mktime().  An adjustment is required for leap years before march, but they didn't match the leap year correctly,